apps/web/src/app/(authenticated)/profile/components/user-info/user-info.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Avatar, Text, Group } from '@mantine/core';
+import { useTranslations } from 'next-intl';
+import { type MeQuery } from '@/graphql/generated/schema-server';
+import classes from './user-info.module.scss';
+
+interface PropsType {
+  userDetails: MeQuery['me'];
+}
+export function UserInfo(props: PropsType): React.ReactNode {
```

Comment:
```suggestion
export function UserInfo({ userDetails }: PropsType): React.ReactNode {
```
seems that it could make the code a bit more readable